src/video/videoframe.cpp

```diff
@@ -417,8 +417,8 @@ bool VideoFrame::FrameBufferKey::operator!=(const FrameBufferKey& other) const
  */
 size_t VideoFrame::FrameBufferKey::hash(const FrameBufferKey& key)
 {
-    std::hash<int> intHasher;
```

Comment:
Not relevant for this pr, but this is likely an identity function.

Reply:
bool probably too

Reply:
Removed.